utils/directory.go

```diff
@@ -76,5 +77,8 @@ func TempDir(p string) string {
 
 // ServicedLogDir gets the serviced log directory
 func ServicedLogDir() string {
-	return "/var/log/serviced"
+	if os.Getenv("SERVICED_LOG_PATH") != "" {
```

Comment:
you don't need this if condition. `config.GetOptions().LogPath` will give you the right path regardless of whether the environment variable was specified or if the default is used. The change you made to `cli/api/options.go` takes care of that for you.